### PR TITLE
Add global styles and change some pages styles

### PR DESCRIPTION
**Pull request by upsu90**
- Added global styles (margins, headers, font)
- Changed material UI colors
- Fixed some code (float -> flex)

### CODE REVIEW

File: web-client/src/components/Account/AboutProfil.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+export default function AboutProfil(props) {
+  const style = makeStyles({
+    main: {
+      display: "flex",
+      justifyContent: "center",
+      alignItems: "center",
+      marginBottom: "40px",
+    },
+    photo: {
+      height: 150,
```
Comment (JustD4nTe) [inline web-client/src/components/Account/AboutProfil.jsx]: We're writing pixel values in quotes

**Review by JustD4nTe — APPROVED** [review]

File: web-client/src/components/Account/AboutProfil.jsx (hunk 1 above)
Comment (upsu90) [reply web-client/src/components/Account/AboutProfil.jsx]: I know :c

File: web-client/src/components/Navigation.jsx [hunk 2]
```diff
@@ -6,97 +6,66 @@ import Box from "@material-ui/core/Box";
...
     },
     logout: {
-      float: "right",
       fontSize: "12px",
+      textAlign: "right",
       color: "#767676",
-      "&:hover": {
-        cursor: "pointer",
-      },
+      lineHeight: "1em",
```
Comment (Kwolik) [inline web-client/src/components/Navigation.jsx]: Maybe add cursor : "pointer"?

**Review by Kwolik — APPROVED** [review]

File: web-client/src/components/Navigation.jsx (hunk 2 above)
Comment (upsu90) [reply web-client/src/components/Navigation.jsx]: done 👍

**Review by KamilDonda — APPROVED** [review]

**Pull request merged**